tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/deep-config/test-suite-a/testnamespace-namespace/testclass-class/testclassmethod-method.html

```diff
@@ -4,49 +4,31 @@
     <meta charset="utf-8">
   </head>
   <body>
-    <section>
-      <h1>testClassMethod</h1>
-      <section>
-        <p><a href="/">Packages</a> > <a href="/test-suite-a/">test-suite-a</a> > <a href="/test-suite-a/testnamespace-namespace/">TestNamespace</a> > <a href="/test-suite-a/testnamespace-namespace/testclass-class/">TestClass</a> > <a href="/test-suite-a/testnamespace-namespace/testclass-class/testclassmethod-method">testClassMethod(testParameter)</a></p>
-      </section>
-      <section>
-        <p>Test class method</p>
-      </section>
-      <section>
-        <h2 id="testclassmethod-signature">Signature</h2>
-        <pre><code class="language-typescript">testClassMethod(testParameter: string): Promise&#x3C;string>;
-</code></pre>
-      </section>
-      <section>
-        <h2 id="testclassmethod-parameters">Parameters</h2>
-        <table>
-          <thead>
-            <tr>
-              <th>Parameter</th>
-              <th>Type</th>
-              <th>Description</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <td>testParameter</td>
-              <td>string</td>
-              <td>A string</td>
-            </tr>
-          </tbody>
-        </table>
-      </section>
-      <section>
-        <h2 id="testclassmethod-returns">Returns</h2>
-        <p>A Promise</p>
-        <p><strong>Return type</strong>: Promise&#x3C;string></p>
-      </section>
-      <section>
-        <h2 id="testclassmethod-throws">Throws</h2>
-        <p>An Error when something happens for which an error should be thrown. Except in the cases where another kind of error is thrown. We don't throw this error in those cases.</p>
-        <p>A different kind of error when a thing happens, but not when the first kind of error is thrown instead.</p>
-        <p>😁</p>
-      </section>
-    </section>
+    <h1>testClassMethod</h1>
+    <p><a href="/">Packages</a> > <a href="/test-suite-a/">test-suite-a</a> > <a href="/test-suite-a/testnamespace-namespace/">TestNamespace</a> > <a href="/test-suite-a/testnamespace-namespace/testclass-class/">TestClass</a> > <a href="/test-suite-a/testnamespace-namespace/testclass-class/testclassmethod-method">testClassMethod(testParameter)</a></p>
+    <p>Test class method</p><h2 id="testclassmethod-signature">Signature</h2>
+    <pre><code class="language-typescript">testClassMethod(testParameter: string): Promise&#x3C;string>;
+</code></pre><h2 id="testclassmethod-parameters">Parameters</h2>
```

Comment:
Can we make it so the `<h2>` renders on a new line?

Reply:
That said, it occurs to me that I can just delete this entire rendering layer now too... If users want to render as HTML, they can grab our `mdast` output and render it how they want...